.eslintrc

```diff
@@ -2,5 +2,8 @@
   extends: "lob",
   globals: {
     expect: false
+  },
+  rules: {
```

Comment:
I included "function" keyword before "(qb)" which was throwing me a lint error which is why i excluded this rule. But i realized the mistake and have fixed this now.